src/main/kotlin/eu/gillissen/rider/usersecrets/OpenUserSecretsAction.kt

```diff
@@ -7,6 +7,7 @@ import com.intellij.openapi.fileEditor.FileEditorManager
 import com.intellij.openapi.project.DefaultProjectFactory
 import com.intellij.openapi.project.Project
 import com.intellij.openapi.vfs.LocalFileSystem
+import com.jediterm.terminal.ui.UIUtil
```

Comment:
Why import a class from JediTerm?
The `isWindows` method in that project isn't that different from mine, and it seems unnecessary to add a dependency on an unrelated project just for that method.

I suggest copying the implementation instead, if there's no similar method in the `com.intellij` namespace

Reply:
Seems like an Alt+Enter gone wrong. Pushed a fix for this.